### PR TITLE
Add Xavier Pigeon as a new bagger with 3 sessions.

### PR DESCRIPTION
**Pull request by xapn**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -1,5 +1,90 @@
...
+        'Expert Méthode & Qualité (Freelance) | auteur de GearsOfTesting.org | développeur de TestAsYouThink.org',
+      picture:
+        'https://gravatar.com/userimage/29545108/d3a9e2ed3d93ef53aff440e053e0faed.jpeg',
+      websites: [
+        {
+          name: 'Website',
+          url: 'https://xavierpigeon.com/',
+        },
+        {
+          name: 'Chrysocode',
```
Comment (dadoonet) [inline baggers.js]: Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

File: baggers.js [hunk 2]
```diff
@@ -1,5 +1,90 @@
...
+          url: 'https://chrysocode.io/',
+        },
+        {
+          name: 'GOST (Gears Of Software Testing)',
```
Comment (dadoonet) [inline baggers.js]: Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

File: baggers.js [hunk 3]
```diff
@@ -1,5 +1,90 @@
...
+          url: 'https://gearsoftesting.org/',
+        },
+        {
+          name: 'TestAsYouThink Project',
```
Comment (dadoonet) [inline baggers.js]: Please use only names listed in the guide: https://github.com/brownbaglunch/bblfr_data/#websites

File: baggers.js [hunk 4]
```diff
@@ -1,5 +1,90 @@
 var data = {
   speakers: [
+    {
```
Comment (dadoonet) [inline baggers.js]: Please move yourself at the end of the list.

**Review by dadoonet — CHANGES_REQUESTED** [review]
I left some comments. Could you please update your PR?

File: baggers.js (hunk 1 above)
Comment (xapn) [reply baggers.js]: Done.

File: baggers.js (hunk 2 above)
Comment (xapn) [reply baggers.js]: Done.

File: baggers.js (hunk 3 above)
Comment (xapn) [reply baggers.js]: Done.

File: baggers.js (hunk 4 above)
Comment (xapn) [reply baggers.js]: Done.

**Review by dadoonet — APPROVED** [review]

**Pull request merged**